### PR TITLE
updating links and removing broken links.

### PR DESCRIPTION
**Pull request by yeungn**

### CODE REVIEW

File: diy-guide.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Depending on the homogeneity of the group, you are may get a lot of repeat quest
 
 ## See Also
 
-* Mozilla Webmaker [Event Guides](https://webmaker.org/event-guides)
+* <s>Mozilla Webmaker [Event Guides]</s>
```
Comment (afeld) [inline diy-guide.md]: Looks like this may have moved to https://teach.mozilla.org/events/resources/#event-guides.

File: related-groups.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ title: Related Groups
 * [CoderDojo](http://coderdojo.com/) (programming classes for kids)
 * [MOOC meetups](http://moocs.meetup.com/)
 * [Meetup](http://www.meetup.com/find/?categories=34)
-* [RailsBridge](http://workshops.railsbridge.org/) (workshops for women)
+* <s>[RailsBridge](workshops for women)</s>
```
Comment (afeld) [inline related-groups.md]: Let's link to http://www.railsbridge.org/ instead.

**Comment by afeld** [comment]
Such a bummer all that content is gone...all of the afeld.me links are from my old blog, which was taken down before I could get the content from it :cry: 

Just to reduce confusion, would you mind removing all of the broken links, rather than doing a strikethrough? Thanks for doing all of this!

**Comment by yeungn** [comment]
updated. happy to contribute.

**Comment by afeld** [comment]
Cool, thanks for doing this!

**Pull request merged**